src/telcameras_v2/models.py

```diff
@@ -12,8 +12,8 @@ class Observation(models.Model):
     latitude = models.DecimalField(max_digits=16, decimal_places=13)
     longitude = models.DecimalField(max_digits=16, decimal_places=13)
     interval = models.SmallIntegerField()                   # e.g. 60     # seconds that this message spans
-    timestamp_message = models.DateTimeField()
-    timestamp_start = models.DateTimeField(db_index=True)
+    timestamp_message = models.DateTimeField()              # Timestamp of the message, not of of when the data was recorded
```

Comment:
Hoeft niet aangepast te worden, maar dit soort comments kan je ook in de `help_text` van de field zetten.